### DISCUSSION TITLE
Clutter remove

### DISCUSSION THREAD

**Discussion by pikuljena** [ARM-DOE/pyart#1438] [Q&A]
how can i remove geographical clutter ? i am using a polarimetric radar having only 6 radarproducts i.e. dBZ,ZDR,PHIDP,RHOHV,WIDTH,VEL

**Comment by zssherman** [comment]
@pikuljena In the past for buildings etc, we've used radar clutter based on a mean and std as the clutter returns had high returns consistently. https://github.com/EVS-ATMOS/cmac2.0/blob/main/cmac/radar_clutter.py

For land or mountains, there is some beam blockage calculations you can do using wradlib etc:
https://github.com/EVS-ATMOS/cmac2.0/blob/83a3aab8c7caf22ba6743fa487fc05c9786ddb2e/cmac/cmac_processing.py#L444

Besides that, Py-ART has some correction functions found here:
https://github.com/ARM-DOE/pyart/tree/main/pyart/correct

But not all target clutter etc

**Comment by wxwolff-home** [comment]
The GPM GV group has released GVRadar that has lots of tools for quality controlling data. https://github.com/search?q=gvradar&type=repositories

**Reply by mgrover1** [reply]
Thanks for sharing here!!
